src/condor_utils/file_transfer.cpp

```diff
@@ -2021,7 +2035,7 @@ FileTransfer::Download(ReliSock *s, bool blocking)
 	Info.success = true;
 	Info.in_progress = true;
 	Info.xfer_status = XFER_STATUS_UNKNOWN;
-	Info.num_cedar_files = 0;
+	Info.stats = ClassAd();
```

Comment:
The Clear() method is more efficient.
```suggestion
	Info.stats.Clear();
```